tests/integration/dumpMetadata/helpers/db.ts

```diff
@@ -0,0 +1,16 @@
+import { container } from 'tsyringe';
+import { Connection, EntityTarget, Repository } from 'typeorm';
+
+import { DumpMetadata } from '../../../../src/dumpMetadata/models/dumpMetadata';
+import { createMultipleFakeDumpsMetadata } from '../../../helpers';
+
+export const getRepositoryFromContainer = <T>(target: EntityTarget<T>): Repository<T> => {
+  const connection = container.resolve(Connection);
+  return connection.getRepository<T>(target);
+};
```

Comment:
this function is only used once, no need to be a separate function.

Reply:
using this function two times :smile_cat: